.github/workflows/bat.yml

```
@@ -79,24 +92,48 @@ jobs:
           set -e
           grep "building" buildlog.txt
           grep "testing" buildlog.txt
+          grep "tests" buildlog.txt
+          grep "terse" buildlog.txt
           grep "deploying" buildlog.txt
           grep "checking" buildlog.txt
           rm buildlog.txt
 
+      - name: Run build with task args
+        uses: ./
+        with:
+          tasks: test("myTests",OutputDetail="concise")
```

Maybe put a space in here to make sure we aren't splitting on spaces.

```suggestion
          tasks: test("myTests", OutputDetail="concise")
```